### ISSUE TITLE
Custom Input doesn't work

### ISSUE THREAD

**Issue by nicedevil007** [bbindreiter/node-red-contrib-alexa-remote2-applestrudel#29]
Hi at all,

maybe I didn't understand everything right from the description but I can't get the node to work with the custom command:

I always get this: `invalid sequence node: "{"type":"regular","text":"Hello!","volume":50,"devices":["Echo Studio"]}" expected layout: "{"type":"","payload":{}}"`

And I setup my function node like this:

```
msg = {
        type: 'speakAtVolume',
        payload: {
            type: 'regular',
            text: 'Hello!',
            volume: 50,
            devices: ['Echo Studio']
        }
}

return msg;
```

I would say that I did everything like it is described here?
![image](https://user-images.githubusercontent.com/17103076/143769257-b3438ae3-d496-4f8d-864a-cec2c45eb993.png)

Oh quick site note: the link in the description should leading to this github repo here instead of the old remote2 right?

**Comment by bbindreiter** [comment]
You've got to put your json payload into any attribute of `msg`, so for example in your function node:

```
msg.something = {
        type: 'speakAtVolume',
        payload: {
            type: 'regular',
            text: 'test',
            volume: 25,
            devices: ['Echo Office']
        }
}

return msg;
```

And then use `msg.something` in the routine node:
![image](https://user-images.githubusercontent.com/3252801/143777555-183b4d3a-1965-454f-865a-b83865b99736.png)

**Comment by nicedevil007** [comment]
Ty... that made it working!

Should be cleared up in the description of the node or?

**Comment by nicedevil007** [comment]
@bbindreiter Hey, I tested it with different situations that I was using with ioBroker before. Unfortunately the execution of a routine doesn't work.

I have something like this `msg.custom = { type: 'routine', payload: { routine: '4b5c315c-8de7-4ef7-9x91-bead5fda89d5'}}
return msg;`

This is the ID of my routine, I also tryed it with the routines name (and yes I made it case sensitive).

It always tells me this: 

![image](https://user-images.githubusercontent.com/17103076/143780743-a068ab6a-f274-4044-8258-00f6ce6a38aa.png)
